src/app/app.config.ts

```diff
@@ -0,0 +1,3 @@
+export const Config = {
```

Comment:
i was thinking this. i could create an interface so when its used, the type should bubble up